public/locales/en/components.json

```
@@ -0,0 +1,135 @@
+    "reload-button-label": "Reload page"
+  },
+  "filter-list": {
+    "search-results": {
+      "empty-results": "No results found. Try another search!"
+    },
+    "search-settings-menu": {
+      "no-selection-tooltip": "Nothing is selected. Search will be empty.",
+      "dropdown-menu-label": "Search in"
+    },
+    "filter-list-input-field": {
+      "cancel-button-aria-label": "Clear search input",
+      "show-more-button": {
+        "less": "Less",
+        "more": "More"
+      }
+    }
+  },
+  "friction-controls": {
+    "rotation-label": "Rotation",
+    "zoom-label": "Zoom",
+    "roll-label": "Roll"
+  },
+  "friction-controls-info": {
+    "heading": "Friction controls",
+    "description": "Enable or disable rotation, zoom, and roll friction. If checked, applies friction to camera movement."
+  },
+  "info-box": {
+    "aria-label": "More information"
+  },
+  "input": {
+    "numeric-input": {
+      "error-label": "Value outside range [{{min}}, {{max}}]"
+    },
+    "step-control-button": {
+      "aria-label": {
+        "increment": "Increment value",
+        "decrement": "Decrement value"
+      }
+    }
+  },
+  "node-navigation-button": {
+    "jump": {
+      "title": "Jump to",
+      "tooltip": "Teleport to object using a fade transition"
+    },
+    "jump-geo": {
+      "title": "Jump to Geo",
+      "tooltip": "Teleport to position using a fade transition"
+    },
+    "focus": {
+      "title": "Focus",
+      "tooltip": "Focus the object (set as anchor). Hold <0>Shift</0> when clicking to focus without retargeting"
+    },
+    "fly": {
+      "title": "Fly to",
+      "tooltip": "Trigger a flight to the object. Hold <0>Shift</0> when clicking to instantly teleport/jump"
+    },
+    "fly-geo": {
+      "title": "Fly to Geo",
+      "tooltip": "Trigger a flight to the position. Hold <0>Shift</0> when clicking to instantly teleport/jump"
+    },
+    "frame": {
+      "title": "Zoom to / Frame",
+      "tooltip": "Frame the object by moving the camera in a straigt line and rotate towards it. Hold <0>Shift</0> when clicking to do it instantaneously"
+    }
+  },
+  "property": {
+    "list-property": {
+      "double-list-placeholder-text": "number1, number2, ...",
+      "int-list-placeholder-text": "integer1, integer2, ...",
+      "string-list-placeholder-text": "item1, item2, ..."
+    },
+    "option-property": {
+      "aria-label": "{{guiName}} option input",
+      "placeholder": {
+        "options": "Choose an option",
+        "no-options": "No options were loaded"
+      }
+    },
+    "selection-property": {
+      "aria-label": "{{guiName}} multi-select",
+      "placeholder-empty-field": "No selection"
+    },
+    "string-property": {
+      "aria-label": "{{guiName}} input"
+    },
+    "property-label": {
+      "tooltip-label": "This property is read-only, meaning that it's not intended to be changed.",
+      "label": "(Read-only)"
+    }
+  },
```

How about removing "property" from the subkeys here?
```suggestion
  "property": {
    "list": {
      "double-list-placeholder-text": "number1, number2, ...",
      "int-list-placeholder-text": "integer1, integer2, ...",
      "string-list-placeholder-text": "item1, item2, ..."
    },
    "option": {
      "aria-label": "{{guiName}} option input",
      "placeholder": {
        "options": "Choose an option",
        "no-options": "No options were loaded"
      }
    },
    "selection": {
      "aria-label": "{{guiName}} multi-select",
      "placeholder-empty-field": "No selection"
    },
    "string": {
      "aria-label": "{{guiName}} input"
    },
    "label": {
      "tooltip-label": "This property is read-only, meaning that it's not intended to be changed.",
      "label": "(Read-only)"
    }
  },
```